package.json

```diff
@@ -44,13 +44,13 @@
   },
   "devDependencies": {
     "benchmark": "^2.1.4",
-    "bufferutil": "^4.0.1",
-    "eslint": "^7.2.0",
+    "bufferutil": "^4.0.3",
+    "eslint": "^7.21.0",
     "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-prettier": "^3.0.1",
-    "mocha": "^7.0.0",
-    "nyc": "^15.0.0",
-    "prettier": "^2.0.5",
-    "utf-8-validate": "^5.0.2"
+    "eslint-plugin-prettier": "^3.3.1",
+    "mocha": "^8.3.0",
```

Comment:
Sorry but this is not acceptable. `mocha@8` does not work on Node.js v8.x

Reply:
Since 8 is EOL. Maybe leave this to next semver-major. Also, it's just a devDependency,not affect too much.

Reply:
Yes, it will be updated when/if we will drop support for Node.js v8.x